internal/cron/cron_test.go

```diff
@@ -233,7 +233,7 @@ func TestSeries(t *testing.T) {
 			"0 0 0 * * 1/2 *",
 			time.Date(2023, 12, 31, 23, 59, 0, 0, time.UTC),
 			time.Date(2024, 1, 31, 0, 0, 0, 0, time.UTC),
-			18,
+			14,
```

Comment:
Just wondering, why did these tests change?

Reply:
They changed due to differences in how the cron implementation behaves. The old implementation treated Sunday as both a 0 and a 7, so it was both even and odd. The new implementation just treats it as a 0.